### PR TITLE
feat: supports paragon theme configuration

### PR DESCRIPTION
**Pull request by andrey-canon**
## Description

This allows to use paragon theme configurations or interactive color feature

## Testing instructions
1. Set interactive color
```
MFE_CONFIG.update({
    "CUSTOM_PRIMARY_COLORS": {
      "pgn-color-primary-base": "#AA0000"
    },
})
```
2. Check stats, feedback and main page
3. Set paragon configuration
```
MFE_CONFIG.update({
    "PARAGON_THEME_URLS": {
        "core": {
            "url": "https://cdn.jsdelivr.net/combine/npm/@edx/paragon@22.0.0-alpha.13/styles/css/themes/light/utility-classes.min.css,npm/@edx/paragon@22.0.0-alpha.13/dist/core.min.css"
        },
        "defaults": {
            "light": "light"
        },
        "variants": {
            "light": {
                "url": "https://css-varsify.s3.amazonaws.com/public/a9959998-0bab-4447-ada5-6819866195f3.css"
            }
        }
    },
})
```
The tenant setting is required
`"css_overrides_file_extra": "https://css-varsify.s3.amazonaws.com/public/a9959998-0bab-4447-ada5-6819866195f3.css"`

4. Check stats, feedback and main page

##With interactive color
image
image
image

## With paragon configuration
image
image
image

## Checklist for Merge

- [ ] Tested in a remote environment
- [ ] Updated documentation
- [ ] Rebased master/main
- [ ] Squashed commits

### CODE REVIEW

**Review by johanseto — COMMENTED** [review]
LGTM. It looked very good now working with css vars. Great job with webpack.
Peek 2024-03-15 11-33


Translation seems working for stats,
Screenshot from 2024-03-15 13-32-35

but I don't know if you have a similar problem with feedback carrousel in ar.
Peek 2024-03-15 13-33

File: eox_nelp/course_experience/frontend/src/components/FeedbackCarousel/index.scss [hunk 1]
```diff
@@ -1,7 +1,6 @@
 @import "~@edx/brand/paragon/fonts";
 @import "~@edx/brand/paragon/variables";
 @import "~@edx/paragon/styles/scss/core/core";
-@import "~@edx/brand/paragon/overrides";
```
[the remark this answers is not in the document]
Comment (andrey-canon) [reply eox_nelp/course_experience/frontend/src/components/FeedbackCarousel/index.scss:4]: This removes the default values, so the css files won't be overridden by these

**Comment by andrey-canon** [comment]
@johanseto I have fixed the ar feedback view, could you please review it again ?

**Comment by johanseto** [comment]
> @johanseto I have fixed the ar feedback view, could you please review it again ?

Now is working, awesome 
Peek 2024-04-01 12-17

**Review by johanseto — APPROVED** [review]

**Pull request merged**